### PR TITLE
add Meetups Translation

### PR DESCRIPTION
**Pull request by RamirezAlex**
@carburo please review

### CODE REVIEW

File: content/community/meetups.md [hunk 1]
```diff
@@ -35,85 +35,85 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
...
-## Ireland {#ireland}
-* [Dublin](https://www.meetup.com/ReactJS-Dublin/)
+## Irlanda {#ireland}
+* [Dublín](https://www.meetup.com/ReactJS-Dublin/)
 
 ## Israel {#israel}
 * [Tel Aviv](https://www.meetup.com/ReactJS-Israel/)
 
-## Netherlands {#netherlands}
+## Paises Bajos {#netherlands}
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
## Países Bajos {#netherlands}
```

File: content/community/meetups.md [hunk 2]
```diff
@@ -35,85 +35,85 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
...
+## Escocia (UK) {#scotland-uk}
+* [Edinburgo](https://www.meetup.com/React-Scotland/)
 
-## Spain {#spain}
+## España {#spain}
 * [Barcelona](https://www.meetup.com/ReactJS-Barcelona/)
 
-## Sweden {#sweden}
+## Suecia {#sweden}
 * [Goteborg](https://www.meetup.com/ReactJS-Goteborg/)
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
* [Gotemburgo](https://www.meetup.com/ReactJS-Goteborg/)
```

File: content/community/meetups.md [hunk 3]
```diff
@@ -127,10 +127,10 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
 * [Los Angeles, CA - ReactJS](https://www.meetup.com/socal-react/)
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
* [Los Ángeles, CA - ReactJS](https://www.meetup.com/socal-react/)
```

File: content/community/meetups.md [hunk 4]
```diff
@@ -127,10 +127,10 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
...
 * [Los Angeles, CA - React Native](https://www.meetup.com/React-Native-Los-Angeles/)
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
* [Los Ángeles, CA - React Native](https://www.meetup.com/React-Native-Los-Angeles/)
```

File: content/community/meetups.md [hunk 5]
```diff
@@ -35,85 +35,85 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
...
+* [Varsovia](https://www.meetup.com/React-js-Warsaw/)
 
 ## Portugal {#portugal}
-* [Lisbon](https://www.meetup.com/JavaScript-Lisbon/)
+* [Lisboa](https://www.meetup.com/JavaScript-Lisbon/)
 
-## Scotland (UK) {#scotland-uk}
-* [Edinburgh](https://www.meetup.com/React-Scotland/)
+## Escocia (UK) {#scotland-uk}
+* [Edinburgo](https://www.meetup.com/React-Scotland/)
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
* [Edimburgo](https://www.meetup.com/React-Scotland/)
```

File: content/community/meetups.md [hunk 6]
```diff
@@ -35,85 +35,85 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
...
-* [Dublin](https://www.meetup.com/ReactJS-Dublin/)
+## Irlanda {#ireland}
+* [Dublín](https://www.meetup.com/ReactJS-Dublin/)
 
 ## Israel {#israel}
 * [Tel Aviv](https://www.meetup.com/ReactJS-Israel/)
 
-## Netherlands {#netherlands}
+## Paises Bajos {#netherlands}
 * [Amsterdam](https://www.meetup.com/React-Amsterdam/)
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
* [Ámsterdam](https://www.meetup.com/React-Amsterdam/)
```

File: content/community/meetups.md [hunk 7]
```diff
@@ -35,85 +35,85 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
...
+* [Medellín](https://www.meetup.com/React-Medellin/)
 
-## Denmark {#denmark}
+## Dinamarca {#denmark}
 * [Aalborg](https://www.meetup.com/Aalborg-React-React-Native-Meetup/)
 * [Aarhus](https://www.meetup.com/Aarhus-ReactJS-Meetup/)
 
-## England (UK) {#england-uk}
+## Inglaterra (UK) {#england-uk}
 * [Manchester](https://www.meetup.com/Manchester-React-User-Group/)
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
* [Mánchester](https://www.meetup.com/Manchester-React-User-Group/)
```

File: content/community/meetups.md [hunk 8]
```diff
@@ -1,25 +1,25 @@
...
-## Belgium {#belgium}
-* [Belgium](https://www.meetup.com/ReactJS-Belgium/)
+## Bélgica {#belgium}
+* [Bélgica](https://www.meetup.com/ReactJS-Belgium/)
 
-## Brazil {#brazil}
+## Brasil {#brazil}
 * [Belo Horizonte](https://www.meetup.com/reactbh/)
 * [Curitiba](https://www.meetup.com/pt-br/ReactJS-CWB/)
 * [Rio de Janeiro](https://www.meetup.com/pt-BR/React-Rio-de-Janeiro/)
```
Comment (carburo) [inline content/community/meetups.md]:
```suggestion
* [Río de Janeiro](https://www.meetup.com/pt-BR/React-Rio-de-Janeiro/)
```

**Comment by RamirezAlex** [comment]
Thank you @carburo, I made the suggested changes.

**Comment by carburo** [comment]
Thank you!

**Pull request merged**